src/software/backend/radio_backend.cpp

```diff
@@ -33,5 +33,15 @@ void RadioBackend::receiveWorld(World world)
     Subject<World>::sendValueToObservers(world);
 }
 
+void RadioBackend::receiveRobotStatus(RobotStatus robot_status)
+{
+    SensorMsg sensor_msg;
+    TbotsRobotMsg robot_msg        = convertRobotStatusToTbotsRobotMsg(&robot_status);
+    TbotsRobotMsg* added_robot_msg = new TbotsRobotMsg();
+    added_robot_msg                = sensor_msg.add_tbots_robot_msg();
+    added_robot_msg                = &robot_msg;
+    Subject<SensorMsg>::sendValueToObservers(sensor_msg);
```

Comment:
I think the best way to do this is `convertRobotStatusToTbotsRobotMsg` returns a `std::unique_ptr` and then we have
```
added_robot_msg = robot_msg.release()
```
Also, `TbotsRobotMsg* added_robot_msg = new TbotsRobotMsg();` can be removed

Reply:
Done.

Reply:
looks like it didn't change yet?

Reply:
Apologies, it's been applied now.